spanner/google/cloud/spanner_v1/keyset.py

```diff
@@ -38,14 +38,13 @@ class KeyRange(object):
     """
     def __init__(self, start_open=None, start_closed=None,
                  end_open=None, end_closed=None):
-        if not any([start_open, start_closed, end_open, end_closed]):
-            raise ValueError("Must specify at least a start or end row.")
+        if ((start_open is None and start_closed is None)
+            or (start_open is not None and start_closed is not None)):
```

Comment:
Sorry I didn't mention this before, but continuations like this should be indented extra so it's not confused with the line below.

```python
if ((start_open is None and start_closed is None)
        or (start_open is not None and start_closed is not None)):
    raise ValueError('Specify exactly one of start_closed or start_open')
```

Reply:
no problem.  
changed.